tests/python/test_household.py

```diff
@@ -0,0 +1,377 @@
+import pytest
+import json
+from unittest.mock import MagicMock, patch
+from sqlalchemy.engine.row import LegacyRow
+
+from policyengine_api.routes.household_routes import household_bp
+from policyengine_api.services.household_service import HouseholdService
+
+from tests.fixtures.household_fixtures import (
+    SAMPLE_HOUSEHOLD_DATA,
+    SAMPLE_DB_ROW,
+    mock_database,
+    mock_hash_object,
+)
+
+
+class TestGetHousehold:
```

Comment:
**nit/nonblocking**: I have a mild preference for testing the blueprint (I.e. specifically wrap the blueprint in an app and test it) instead of the full app for all tests. Why tightly couple all that stuff if you don't need to?

Reply:
That makes sense. Previously, we had no blueprints, and some of our earliest tests were merely reactions to particular calculations failing, and thus were essentially targeted e2e's or integration tests. Moving forward, I think that would make a lot of sense.